pennylane/tape/__init__.py

```diff
@@ -70,7 +71,14 @@ def enable_tape():
     if _mock_stack:
         return
 
-    mocks = [mock.patch("pennylane.qnode", qnode), mock.patch("pennylane.QNode", QNode)]
+    mocks = [
+        mock.patch("pennylane.qnode", qnode),
+        mock.patch("pennylane.QNode", QNode),
+        mock.patch("pennylane.expval", measure.expval),
+        mock.patch("pennylane.var", measure.var),
+        mock.patch("pennylane.probs", measure.probs),
+        mock.patch("pennylane.sample", measure.sample),

```

Comment:
Make sure that the measurement functions are changed *globally*, not just within tape contexts, when `qml.enable_tape()` is called.